numpy/random/mtrand/mtrand.pyx

```diff
@@ -4653,14 +4653,15 @@ cdef class RandomState:
 
     def dirichlet(self, object alpha, size=None):
         """
-        dirichlet(alpha, size=None)
+        dirichlet (alpha, size=None)
```

Comment:
Did you mean to put a space here?  Normally this should match the function declaration omitting self.

Reply:
No, I did not do this on purpose. I just wanted to edit the documentation.